### ISSUE TITLE
What is a Collection?

### ISSUE THREAD

**Issue by kinlane** [postman-open-technologies/collections#9]
Landing page, blog posts, videos, licensing, schema, and other resources that showcase what is possible with a collection, and helps you understand how to put to work.

**Comment by kinlane** [comment]
Can we do this as a landing page, and become the single canonical URL for everything collection, helping new users and advanced users find what they need.

**Comment by Gbahdeyboh** [comment]
Yes, that's definitely a great suggestion! It will serve as a north star for everything collection related and new resources that are created elsewhere can be linked(blog posts, videos, learning center, etc) into that single page making us have everything about collections all in a single place.
